src/ray/common/ray_config_def.h

```diff
@@ -916,10 +916,8 @@ RAY_CONFIG(int64_t, py_gcs_connect_timeout_s, 30)
 // Costs an extra RPC.
 // TODO(vitsai): Remove this flag
 RAY_CONFIG(bool, enable_reap_actor_death, true)
-// The number of sockets between object manager.
-// The higher the number the higher throughput of the data
-// transfer it'll be, but it'll also user more sockets and
-// more CPU resources.
+
```

Comment:
Can we make the code to do what it is intended to do instead? (i.e. to actually control the number of underlying connections)?

Reply:
No, because it doesn't guarantee to have better performance in all cases.

Reply:
Update:
No, because:
- It's a behavior change
- Performance degrades with unnecessary resource usage increases in certain cases.

Reply:
What benefit does the current behavior have if the underlying connection is 1? Should we change the config to really mean the number of underlying connections and change the default value to 1 so no behavior change?

Reply:
> What benefit does the current behavior have if the underlying connection is 1?

What do you mean? Could you please elaborate? I don't understand what you are asking about?

Reply:
> Should we change the config to really mean the number of underlying connections and change the default value to 1 so no behavior change?

Then you need to add another config value, which indicates 4 grpc clients. I don't know what's the benefit.

Reply:
> Should we change the config to really mean the number of underlying connections and change the default value to 1 

I don't know how you could do that also, bu default all stubs share one connection

Reply:
If not, I would suggest we just delete the config and logic entirely...

Reply:
ok, deleted.